README.md

```diff
@@ -28,17 +28,21 @@ To run the latest example notebooks, execute the commands below which will insta
 
 ```console
 $ python -m pip install virtualenv
-$ python -m venv env
-$ source env/bin/activate
+$ python -m venv cleanlab-examples # creates a new venv named cleanlab-examples
```

Comment:
```suggestion
$ python -m venv cleanlab-examples  # creates a new venv named cleanlab-examples
```

Reply:
[nit] just fyi, want to use 2 spaces before comment in [PEP-8](https://peps.python.org/pep-0008/#inline-comments)